src/aleph/vm/orchestrator/chain.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from typing import Dict, Optional, Union
+
+from aleph_message.models import Chain
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class ChainInfo(BaseModel):
+    """
+    A chain information.
+    """
+
+    chain_id: int
+    rpc: str
+    token: str
+    super_token: Optional[str] = None
```

Comment:
Would it not be better to have `token`, `super_token` or both, but not have a duplicate between `token` and `super_token` ? Maybe add `standard_token` and change `token` to be a `@property` :thinking: ?

Reply:
i just push the change request